common/rsyslog_plugin.cpp

```diff
@@ -0,0 +1,161 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <unordered_set>
+#include <regex>
+#include <tuple>
+#include "logger.h"
+#include "json.hpp"
+
+using namespace std;
+using namespace nlohmann;
+
+string parserCfgFile = "{}_parse.rc.json";
```

Comment:
>parserCfgFile [](http://example.com/codeflow?start=7&length=13)

try follow the naming convention / code style in other C++ files. For global variable, use `g_parserCfgFile`. For const, use `LOG_FILE_NAME`.

Another suggestion is not to use the global variables if you can.

Reply:
Done